pyspectral/tests/unittest_helpers.py

```diff
@@ -28,3 +28,21 @@ def assertNumpyArraysEqual(self, other):
         raise AssertionError("Shapes don't match")
     if not np.allclose(self, other):
         raise AssertionError("Elements don't match!")
+
+
+class CustomScheduler(object):
```

Comment:
```suggestion
class CustomScheduler:
```
I was a bit confused by the name of this seeing it used in the test module, can we have a better name? `DaskComputeCounter`? Or have Imissed something?

Reply:
This comes from satpy and pyresample and other packages that have been adapted to dask and needed to check the same kinds of things in tests. It could definitely have a better name, but it might be best to keep the `Scheduler` at the end of the name since that is technically what it is mimicking...I guess just good docstrings would help with this too. Or I think in some other packages we made a context manager function that creates and uses the scheduler in the `dask.config` or you so you don't have to know how to set a scheduler or anything. That function can have a nice use-case-specific name that may be more clear than whatever this class ends up being named.